samcli/commands/init/init_templates.py

```diff
@@ -124,7 +128,9 @@ def get_app_template_location(self, template_directory):
         return os.path.normpath(os.path.join(self._git_repo.local_path, template_directory))
 
     def get_manifest_path(self):
-        return Path(self._git_repo.local_path, self.manifest_file_name)
+        if self._git_repo.local_path and os.path.exists(Path(self._git_repo.local_path, self.manifest_file_name)):
```

Comment:
Instead of `os.path.exists` could we use `Path(..).exists()` for `pathlib` consistency moving forward?